epoch/api/oracle_api_usage.md

```diff
@@ -16,12 +16,18 @@ The following assumes that the node exposes at address `localhost/127.0.0.1` the
 
 ## Oracle management flow
 
-In order to work through the example we also need the (Base58Check-encoded)
-public key of the node. This is easily retrieved from the running node:
+In order to work through the example we need the (Base58Check-encoded)
+public key of the account with positive balance. For simplicity, we will use
+beneficiary account. Public key can be found in configuration file
+( `mining > beneficiary` parameter).
```

Comment:
not sure if whitespace before `mining` is necessary

Reply:
It's not ;)